dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sme/SmeData.kt

```diff
@@ -0,0 +1,86 @@
+package org.dataland.datalandbackend.model.sme
+
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.model.enums.commons.YesNoNa
+import org.dataland.datalandbackend.model.enums.sme.CompanyAgeBracket
+import org.dataland.datalandbackend.model.enums.sme.EnergyEfficiencyBracket
+import org.dataland.datalandbackend.model.enums.sme.EnergyProductionBracket
+import org.dataland.datalandbackend.model.enums.sme.HeatSource
+import org.dataland.datalandbackend.model.enums.sme.Industry
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for the SME framework
+ */
+@DataType("sme")
+data class SmeData(
+    val industry: Industry?,
+
+    val financialYear: Long?,
+
+    val totalRevenue: BigDecimal?,
+
+    val researchAndDevelopmentExpenses: BigDecimal?,
+
+    val energyEfficiencyInvestments: EnergyEfficiencyBracket?,
+
+    val operatingCosts: BigDecimal?,
+
+    val totalAssets: BigDecimal?,
+
+    val yearsSinceFounded: CompanyAgeBracket?,
+
+    val productCategoryHighestSales: String?,
+
+    val productCategoryHighestSalesShareOfSales: BigDecimal?,
+
+    val productCategorySecondHighestSales: String?,
+
+    val productCategorySecondHighestSalesShareOfSales: BigDecimal?,
+
+    val totalAreaCompany: BigDecimal?,
+
+    val totalPowerConsumption: BigDecimal?,
+
+    val totalPowerCosts: BigDecimal?,
+
+    val useOfGreenElectricity: YesNoNa?,
+
+    val heatingEnergyConsumption: BigDecimal?,
+
+    val totalHeatingCosts: BigDecimal?,
+
+    val roomWaterHeating: HeatSource?,
+
+    val shareOwnEnergyProduction: EnergyProductionBracket?,
+
+    val waterSewageCosts: BigDecimal?,
+
+    val wasteDisposalCosts: BigDecimal?,
+
+    val wasteRecyclingRate: BigDecimal?,
+
+    val numberOfEmployees: Long?,
+
+    val numberOfTemporaryWorkers: Long?,
```

Comment:
Is this the same as "number of employees of subcontractors comission"?

Reply:
Ich habe immer die Spalte "Field" aus dem Excel als Namen für die Variablen genommen nicht die "Question" Spalte

Reply:
Ah, ich sehs.